_pages/ebp_insects_agenda.md

```diff
@@ -0,0 +1,34 @@
+**Earth BioGenome meeting 2019 - Insects Session**  **agenda**
```

Comment:
We need to add a header to let Jekyll know what this page is about.

```
---
title: "The Insects breakout session for the Earth BioGenome 2019 meeting"
type: single
permalink: /ebp_insects_agenda
---
```

Reply:
Adding this fixed the linking issue form the main page.

Reply:
Awesome, thanks!